src/pages/settings/Payments/AddPayPalMePage.js

```diff
@@ -36,7 +36,7 @@ class AddPayPalMePage extends React.Component {
         super(props);
 
         this.state = {
-            payPalMeUsername: props.payPalMeData.description,
+            payPalMeUsername: props.payPalMeData.description || '',
```

Comment:
Why do we need  `props.payPalMeData.description`, please clean up other unnecessary codes.